rest/server.js

```diff
@@ -1,46 +1,53 @@
 const express = require("express");
-const { v4: uuidv4 } = require("uuid");
 const bodyParser = require("body-parser");
 const app = express();
-const users = [];
+const { MongoClient } = require("mongodb");
+const ObjectId = require('mongodb').ObjectId; 
+const uri =
+  "mongodb+srv://admin:admin@cluster0.catjmgw.mongodb.net/?retryWrites=true&w=majority";
```

Comment:
move it to environment variables. User credentials like the other tokens should never be stored in repo